remove.js

```diff
@@ -0,0 +1,21 @@
+// select the parent element that holds all the book items
+const bookContainer = document.querySelector('.book-lists');
+
+// define a function to remove a book item
+function removeBook(event) {
+  // get the clicked remove button
+  const removeBtn = event.target;
+  // get the closest ancestor element with the 'book-list' class
+  const bookItem = removeBtn.closest('.book-list');
+  // remove the book item from the DOM
+  bookItem.remove();
+}
```

Comment:
- [ ] At the moment, this remove functionality will only remove the book from the screen and not from the storage. Please kindly ensure it removes the right book from the screen and storage.